app/views/admin/loans/questionnaires/_breakeven_product_row.html.slim

```diff
@@ -3,12 +3,18 @@
 - prefix = "loan_response_set[#{question.attribute_sym}][breakeven][products][]"
 
 tr.ui-state-default data-group="product" class=("hidden" if hidden)
-  td: textarea name="#{prefix}[name]" = product[:name]
+  td: textarea.name name="#{prefix}[name]" = product[:name]
   td: textarea name="#{prefix}[description]" = product[:description]
   td: textarea name="#{prefix}[unit]" placeholder=I18n.t('loan.breakeven.placeholders.unit') = product[:unit]
-  td: input name="#{prefix}[price]" type="number" value=product[:price]
-  td: input name="#{prefix}[cost]" type="number" value=product[:cost]
-  td: input name="#{prefix}[quantity]" type="number" value=product[:quantity]
+  td: input.price name="#{prefix}[price]" type="number" value=product[:price]
+  td: input.cost name="#{prefix}[cost]" type="number" value=product[:cost]
+  td: input.percentage_of_sales name="#{prefix}[percentage_of_sales]" type="number" value=product[:percentage_of_sales]
+  td
+    input.quantity type="number" value=product[:quantity] disabled=true
```

Comment:
I fought with styling a div to look the same for awhile and wasn't able to do it. So I kept them as inputs, but disabled them.